tests/application/admin/test_views.py

```diff
@@ -180,19 +176,19 @@ def test_reset_password_accepts_good_password(app, test_app_client):
     assert page.find("h1").text.strip() == "Password updated"
 
 
-def test_admin_user_can_share_page_with_dept_user(test_app_client, mock_dept_user, mock_admin_user, stub_measure_page):
-
+def test_admin_user_can_share_page_with_dept_user(test_app_client, dept_user, admin_user):
+    measure_page = MeasureVersionFactory(id=100)
```

Comment:
How come this needs a specific id?

Reply:
...because the test fails without it!

I'lll look into it.  Feels like it shouldn't need this probably.

Reply:
For some reason it looks like MeasureVersionFactory() creates *four* MeasureVersions, all with id=0.  This means that `get_measure_from_measure_version_id()` chokes when it tries to get one measure version by ID.

If we specify a non-zero id then we get three MeasureVersions with id 0 and one with the specified id, and the test passes OK.

I'll dig in to why we get four things a bit more, as this feels like FactoryBoy is doing something wrong.

Reply:
OK, it's because the homepage/topic/subtopic MeasureVersions that get created.  Which will go away once we do the tidy up.  So *maybe* I leave the explicit id in here with a TODO to fix it?

Reply:
Ah, we can add a relatively simple patch for this by changing the `id` sequence to do something hacky like `id = factory.Sequence(lambda x: x + 10000)`, `x + 20000`, `x + 30000` for homepage/topic/subtopic factories. Then they'll no longer overlap. This should work until we get rid of them.